.github/workflows/validator.yml

```diff
@@ -93,13 +73,29 @@ jobs:
 
     - name: Download dataset
       run: |
-        git fetch --depth=1 origin git-annex:git-annex # actions/checkout@v2 does a shallow checkout, so it is missing this important branch
+        git fetch --depth=1 origin git-annex:git-annex # actions/checkout@v4 does a shallow checkout, so it is missing this important branch
         git annex init
         git annex get -J8
 
     - name: Checking BIDS compliance
-      run: bids-validator --verbose ./
+      if: always()
+      run: |
+        bids-validator --verbose ./ | tee bids.txt
+        printf "\nShort summary:\n\n"
+        # FIXME: This should be replaced with bids-validator settings: A) throw error on warning or B) warning ignores
+        ! grep "\[WARN\]\|\[ERR\]" bids.txt  # ! -> Reverse error code so "found == error thrown"
+
     - name: Checking acquisition parameters
-      run: sg_params_checker -path-in ./
+      if: always()
+      run: |
+        sg_params_checker -path-in ./ | tee params.txt
+        # FIXME: Checking the output like this should be replaced with SG throwing actual errors
+        ! grep "WARNING" params.txt  # ! -> Reverse error code so "found == error thrown"
+
     - name: Checking data consistency
-      run: sg_check_data_consistency -path-in ./
+      if: always()
+      run: |
+        sg_check_data_consistency -path-in ./ | tee consistency.txt
+        # FIXME: Checking the output like this should be replaced with SG throwing actual errors
+        ! grep "Warning\|Missing" consistency.txt
+        grep "all good" consistency.txt
```

Comment:
Note: The failures on this PR are to be expected, because all 3 of these checks throw warnings/errors.

We should brainstorm on how to best fix these issues!

Reply:
BIDS: The WARNINGs can be ignored. (README can be fixed.)

Reply:
Automated testing was done right before a conference, just for demo purposes, so these warnings aren't actually taken into account for acquisition parameters/data consistency.